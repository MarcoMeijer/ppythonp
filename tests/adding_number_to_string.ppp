print(154 + "2")
print("a" + 543)
