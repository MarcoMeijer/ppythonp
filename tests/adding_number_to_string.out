1542
a543
